src/resonanceReconstruction/rmatrix/legacy/CompoundSystemBase/src/ctor.hpp

```diff
@@ -3,8 +3,8 @@
 /**
  *  @brief Private constructor
  */
-CompoundSystem( std::vector< unresolved::SpinGroup >&& groups,
-                unsigned int lmax ) :
+CompoundSystemBase( std::vector< SpinGroupType >&& groups,
+                    unsigned int lmax ) :
```

Comment:
I may be one of the only individuals in the world who does this, but I'll even write "const" on simple pass-by-value parameters that aren't modified in the function, just to quickly tell someone who's studying the function, that they can know ahead of time that the function doesn't modify even the formal parameter. Obviously, the actual parameter doesn't change either way, because it's pass by value, but the const still tells us something.